src/main/java/net/minecraftforge/common/ForgeConfigSpec.java

```diff
@@ -741,6 +838,30 @@ public boolean test(Object t)
             return result;
         }
 
+        public Component getErrorMessage(Object t) {
+            if (isNumber(t))
+            {
+                Number n = (Number) t;
+                boolean result = ((Number)min).doubleValue() <= n.doubleValue() && n.doubleValue() <= ((Number)max).doubleValue();
```

Comment:
I wonder if instead of having an inverted if statement below we should just have something along the lines of:
```java
if (check from this line)
throw exception valid value
```

Reply:
Maybe, but this code was structured in this form already, I did not want to mess with this too much.